stories/Form.js

```diff
@@ -146,6 +148,61 @@ stories
       size={select('Size', ['', 'small', 'medium', 'large'], '') || undefined}
     />
   ))
+  .add('Combobox', () => (
+    <Combobox
+      inlineListbox={boolean('Inline listbox variant', false)}
+      readOnly={boolean('Read-only variant', false)}
+      error={text('Error', '') || undefined}
+      height={select('Height', [5, 7, 10]) || undefined}
+      id="picklist-1"
+      isDisabled={boolean('Disabled', false) || undefined}
+      hideLabel={boolean('Label hidden', false) || undefined}
+      required={boolean('Required', false) || undefined}
+      items={object('Items', [
```

Comment:
I do not understand how the uncontrolled `Combobox` should work here. This is supposed to be a _seed_, correct? Shouldn't the Combobox handle all selections from then on? Currently, it does not seem like this is the case.

Reply:
The uncontrolled `Combobox` is supposed to only render what's provided via prop, without managing some internal state. So `items`is not really a seed but just what needs to be rendered.

Reply:
The naming is backwards then. You are describing `ControlledCombobox` since it it controlled by props.